features/networking/sctp.feature

```diff
@@ -18,6 +18,13 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[0].name %> |
     Then the step should succeed
     And the pod named "sctpserver" becomes ready
+    And I wait up to 20 seconds for the steps to pass:
+    """
+    When I execute on the "sctpserver" pod:
+      | bash | -c | which nc    |
```

Comment:
Thanks @weliang1 for investigating comment. LGTM except just a nit bit, you have extra 2 white spaces post ```which nc```. if you could fix in all cases. Thanks !

Reply:
@anuragthehatter Remove the extra spaces now. Thanks for reviewing!